<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>537</height>
   </rect>
  </property>
  <widget class="QDockWidget" name="dockWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>270</width>
     <height>228</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Projects</string>
   </property>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTreeView" name="tvProjects">
       <property name="autoExpandDelay">
        <number>10</number>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>270</width>
     <height>228</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Open Issues</string>
   </property>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTableView" name="tvIssues"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>270</width>
     <height>228</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Time Entries</string>
   </property>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTableView" name="tvTimeEntries"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>270</width>
     <height>228</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Comrades</string>
   </property>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTableView" name="tvUsers"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
